Doc/whatsnew/3.7.rst

```diff
@@ -187,6 +187,11 @@ API and Feature Removals
   Python 3.1, and has now been removed.  Use the :func:`~os.path.splitdrive`
   function instead.
 
+* Removed old-deprecated classes ``Plist``, ``Dict`` and ``_InternalDict``
```

Comment:
This is pretty minor, but perhaps we can add Python versions too:

* \`\`Dict\`\` (deprecated in Python 3.0)
* \`\`Dict\`\` (Python 3.0)